net.retrodeck.retrodeck.yml

```diff
@@ -91,7 +91,7 @@ modules:
         echo "Version is $VERSION"
     sources:
       - type: git
-        url: https://github.com/XargonWan/RetroDECK.git
+        url: RDREPO
```

Comment:
Maybe this is a leftover from your previous local tests?

Reply:
I wanted to draw attention to how I got it to build from my repo.

Without RDREPO it would not build. Without manually changing repos here

Need to document what this is better for when it switches back to the main repo or another repo

Reply:
Yes, you're correct, we need to document it or even automate something that if we're not on the main repo it will do some magic.
Can you please open an issue with the detailed situation please?

However to being able to merge this must be fixed for our repo.